google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -113,13 +113,19 @@ public BucketInfo apply(com.google.api.services.storage.model.Bucket pb) {
    */
   public enum PublicAccessPrevention {
     ENFORCED("enforced"),
-    /** Default value for Public Access Prevention */
-    UNSPECIFIED("unspecified"),
+    /**
+     * Default value for Public Access Prevention
+     *
+     * @deprecated use {@link #INHERITED}
+     */
+    @Deprecated
+    UNSPECIFIED("inherited"),
```

Comment:
@shaffeeullah will the service still support unspecified to continue supporting clients that are released?